lib/OptimizelyClient.js

```diff
@@ -429,3 +509,30 @@ OptimizelyClient.prototype.pushVariation = function(options) {
     this.createVariation(options);
 }
 module.exports = OptimizelyClient;
```

Comment:
This is called twice and not at the end. Not a problem per se, just curious as to why.